R/ResamplingSpCVBuffer.R

```diff
@@ -5,6 +5,17 @@
 #' @description Spatial Buffer Cross validation implemented by the `blockCV`
 #' package.
 #'
+#' @note
+#' The `blockcv::buffering` function was written for species distribution modeling with presence-absence data.
+#' However, the default settings allow to conduct a leave-one-out cross validation
+#' for two-class, multi-class and continuous response data, where each observation is one test set.
+#' For each test, all observations outside the buffer around the test observation are included in the training set.
+#'
+#' The parameter `spDataType = PB` and `addBG` are designed for presence-background data in species distribution modelling.
+#' If `spDataType = PB`, test sets are only created for each presence observation (`task\$postive`).
+#' The option `addBG = TRUE` adds the background data inside the buffer to the corresponding test sets.
+#' For each test set, all observations outside the buffer around the test observation are included in the training set.
```

Comment:
Please format to a width of 80. See other PR for instructions.

Reply:
I merged the commits from the other PRs.